### ISSUE TITLE
Make the recipe work better with the autotick bot

### ISSUE THREAD

**Issue by bollwyvl** [conda-forge/spacy-model-en_core_web-feedstock#3]
Ref #2

The current structure of the feedstock doesn't get picked up by the bot when new releases drop. We should either change the recipe, or PR to the bot to make this work better in the future.

**Comment by setu4993** [comment]
+1 on this. Do you have examples of other repositories that use multiple input sources and get auto-updated? We could get inspiration from there.

**Comment by bollwyvl** [comment]
Here's one that has a canary file that the bot understands well (pip), and then a bunch of crazy stuff (rando binaries, but hey they're built by google, what could go wrong?):
https://github.com/conda-forge/python-chromedriver-binary-feedstock/blob/master/recipe/meta.yaml#L9

I think the issue comes from the `sm`, `medium` and `lg` in `conda_build_config.yml`. If it was self-contained, and just generated three outputs instead of one output, it might be more sane...

**Comment by setu4993** [comment]
Interesting. Agree, I think the nested folder structure and `conda_build_config.yml` cause the bot to trip.

Do you know if there's a way to get a variable from the install command used? So, if someone entered `conda install spacy-model-en_core_web_sm`, could we extract `sm` into, say, `size` as `{% set size = get_from_command() %}`. Then, we could just do:
```
url : XXX  # [size==sm]
sha256 : XXX  # [size==sm]
url : XXX  # [size==md]
...
```

**Comment by setu4993** [comment]
^ I attempted to do exactly this by using the `CONFIG` environment variable. Linting checks fail because of line 4, but not sure how else to do it. https://github.com/conda-forge/spacy-model-en_core_web-feedstock/pull/4

**Comment by bollwyvl** [comment]
we need to look at the history of upstream releases. If sm/m/lg always
update at the same time, we should be able to have three sources (with no
selectors) and three outputs, and get rid of the conda_build_config
altogether.

Or, again, try loading it into the bot's test framework, and see what comes
out... Unfortunately don't have the link at present...

**Comment by setu4993** [comment]
Just checked the dates of the latest releases and they do release at the same time.

Hmmm, I'll try that on the PR.

**Comment by setu4993** [comment]
It seems #5 didn't fix this; v2.2.5 of the spacy-models are [released](https://github.com/explosion/spacy-models/releases/tag/en_core_web_sm-2.2.5), but the bot didn't pick it up.

**Comment by setu4993** [comment]
Just tried creating a new [test](https://github.com/setu4993/cf-scripts/commit/2948649db3a53761fb983f58a8c360f18b94f6b9) in the `regro/cf-scripts` repo with the contents of the `meta.yaml` for this project, and it seems to work successfully.

**Comment by setu4993** [comment]
Looks like https://github.com/regro/cf-scripts/pull/711 fixed it and caught 2.2.5 (#6), that was released a couple weeks back :).

Closing the issue.

**Comment by bollwyvl** [comment]
Hooray!
